### PR TITLE
Add codeset example for mlflow/tensorflow

### PR DESCRIPTION
**Pull request by flaviodsr**
The example comes from: https://github.com/mlflow/mlflow/tree/master/examples/tensorflow/tf2

This commit also changes the directory structure for codesets by adding
a sub directory called mlflow and moving the example codeset to it under
a directory named by the kind of training library the codeset uses.

### CODE REVIEW

**Comment by jsuchome** [comment]
do you have the documentation update in (I think) the main repo? It probably mentions the directory structure so it needs to be updated

**Comment by flaviodsr** [comment]
Thanks for pointing that out, I completely missed it. I think this is the one you are referring to: https://github.com/fuseml/docs/pull/11

**Review by jsuchome — APPROVED** [review]

File: codesets/mlflow/sklearn/conda.yaml [hunk 1]
```diff
@@ -3,7 +3,6 @@ channels:
   - defaults
 dependencies:
   - python=3.6
-  - boto3
```
Comment (jsuchome) [inline codesets/mlflow/sklearn/conda.yaml:6]: This is because you've added `boto3` into the image, right?

Comment (flaviodsr) [reply codesets/mlflow/sklearn/conda.yaml:6]: exactly, this was added by us. The original example does not really have this dependency.

**Pull request merged**